1. When running `python python-version/utils/script_which_complete_db.py`, if errors appear during downloads, it is because the links weren't updated according to this guideline: https://docs.google.com/document/d/1WnvA8gr82Y56Zya_UO43fFZF23vPOszR115IxnFleMU/edit

2. After we modify code, this is how we deploy to Docker: https://docs.google.com/document/d/1bKnLH0I2XBak_6vGs6ikKmQfo42bJNex7nqLKaAkO-U/edit

3. If you modify settings and want to reset, and start from 0 all the scans, be aware that URLs are going to change.

4. How to install on Linux/Windows: https://docs.google.com/document/d/1pmJKLgX5FNA-K2KcdYnJ_PzjgTbNOFkXG8mH8JsSa50/edit
- If issue appear that couldn't connect to db, but in other machines work, try deactivating the VPN and test like that. If works, probably another faster country should be selected on VPN.

5. What did we try on VPNs/how to set up VPN properly: https://docs.google.com/document/d/1orvdfSovxJksWHwQdEnf14j0R47t7ujILJcwsCbAj-g/edit